dask/dataframe/core.py

```diff
@@ -869,7 +870,10 @@ def nunique(self):
     @wraps(pd.Series.value_counts)
     def value_counts(self):
         chunk = lambda s: s.value_counts()
-        agg = lambda s: s.groupby(level=0).sum().sort(inplace=False, ascending=False)
+        if LooseVersion(pd.__version__) > '0.16.2':
+            agg = lambda s: s.groupby(level=0).sum().sort_values(ascending=False)
+        else:
+            agg = lambda s: s.groupby(level=0).sum().sort(inplace=False, ascending=False)
```

Comment:
In the future we might find that we want to keep `sort_values` in something like a `dask/dataframe/compatibility.py`.